### ISSUE TITLE
Avatar image covers the content when inspect is opened

### ISSUE THREAD

**Issue by justaskulkarni** [creativecommons/creativecommons.github.io-source#699]
## Description

In https://opensource.creativecommons.org/blog/authors when inspect is opened for any of the author, the avatar image covers the content.

## Reproduction

1. Navigate to https://opensource.creativecommons.org/blog/authors/sara and inspect.
2. The same can be observed for other authors as well. Examples have been attached as screenshots.

## Expectation

Change the width in media queries.

## Screenshots

![opensource_creativecommons_org_blog_authors_sara](https://user-images.githubusercontent.com/98650743/221397953-2c5be116-1a7f-49a5-a088-0e8ea5d36177.jpg)
![image](https://user-images.githubusercontent.com/98650743/221398161-8518a4a3-9cad-4afd-a1a8-c4a7946c32a9.png)

## Resolution

The solution is as simple:
In 
.column.is-2, .column.is-2-tablet {
    flex: none;
    width: 16.66666674%;
  }
Change the width
.column.is-2, .column.is-2-tablet {
    flex: none;
    width: 21.66666674%;
  }
After resolution:
![image](https://user-images.githubusercontent.com/98650743/221398586-65006729-7908-4a8d-9fa6-a73cb7993fba.png)
![image](https://user-images.githubusercontent.com/98650743/221398676-bd4d56cd-3db9-4394-9930-5aa297e31793.png)
- [x] I would be interested in resolving this bug.

**Comment by TimidRobot** [comment]
The proposed fix overrides Bulma styling. Please identify a way to make the authors page more responsive using Bulma or CC Vocabulary features instead of overriding.

**Comment by justaskulkarni** [comment]
@TimidRobot can we add a custom class that sets the width to 21.66666674% as a workaround? 
<div class = "column is-2 custom-class">
and then in it's corresponding CSS we can have 
.custom-class{
width:21.66666674%;
}

**Comment by justaskulkarni** [comment]
I've had another try at it. This time I have used Bulma. 
I figured out that the responsiveness issue was only for tablet screens.
Here are the changes I propose:
Earlier code:
![image](https://user-images.githubusercontent.com/98650743/222236087-e41f70a4-de21-4315-b8a1-62369155acc9.png)
Updated code:
![image](https://user-images.githubusercontent.com/98650743/222236522-8c2ed3e1-a6ed-4075-8a2d-77cdc984563a.png)
is-one-fifth doesn't give the desirable responsiveness, therefore, I went for is-one-quarter.
Responsiveness is achieved as demonstrated below:

https://user-images.githubusercontent.com/98650743/222238059-4cb89a10-f5b8-4d5a-8a9e-e90633fff7a6.mp4
